fastify.js

```diff
@@ -629,6 +644,50 @@ function fastify (options) {
   }
 }
 
+function normalizeOptions (options) {
+  const deprecatedOptionsMap = new Map([
```

Comment:
I would add these settings to `log`:

```
  logger
  requestIdLogLabel
  disableRequestLogging
  requestIdHeader
  genReqId
```

Reply:
Thanks for pointing this out. I will add another field for these